templates/shared_services/certs/Dockerfile.tmpl

```diff
@@ -1,22 +1,25 @@
-FROM python:3.8
+FROM python:3.8-slim-buster
 
 ARG BUNDLE_DIR
 
-RUN apt-get update \
-    && apt-get install -y ca-certificates \
-    && apt-get clean -y && rm -rf /var/lib/apt/lists/*
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Install Azure CLI
+# It's useless to specify azcli version since the mixin installs the latest anyway
```

Comment:
You don't specify the Az CLI version in this dockerfile, but you do in nexus. What is the difference between the porter mixin

Reply:
If the az mixin is installed then it uses the latest version anyway which overrides anything I will specify here. Nexus doesn't install the mixin which makes it possible to put in a version there, but not here...

There's a PR in progress on the mixin to support versions which will enable us to fix this as well.